SimPEG/Regularization.py

```diff
@@ -260,7 +283,23 @@ def aveFy2CC(self):
         :return: averaging from active cell centers to active y-faces
         """
         if getattr(self, '_aveFy2CC', None) is None:
-            self._aveFy2CC = self.Pac.T * self.mesh.aveFy2CC * self.Pafy
+            if self.mesh._meshType == "TREE":
```

Comment:
eventually, I think we should take some steps to simplify this, but for now it is okay